src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -0,0 +1,99 @@
+| [Places Service](https://experienceleague.adobe.com/docs/places/using/home.html) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/places.svg?logo=android&logoColor=white&label=places&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/places) |
+| [Adobe Analytics](../../../solution/adobe-analytics/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/analytics.svg?logo=android&logoColor=white&label=analytics&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/analytics) |
+| [Adobe Analytics - Media Analytics for Audio & Video](../../../solution/adobe-media-analytics/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/media.svg?logo=android&logoColor=white&label=media&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/media) |
+| [Adobe Target](../../../solution/adobe-target/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/target.svg?logo=android&logoColor=white&label=target&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/target) |
+| [Adobe Campaign Standard](../../../solution/adobe-campaign-standard/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaign.svg?logo=android&logoColor=white&label=campaign&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaign) |
+| [Adobe Campaign Classic](../../../solution/adobe-campaign-classic/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic) |
+| [Adobe Audience Manager](../../../solution/adobe-audience-manager/index.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/audience.svg?logo=android&logoColor=white&label=audience&style=flat-square&versionPrefix=3)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/audience) |
+
+## Migrate from Adobe Experience Platform 2.x SDKs for Android
+
+If you have implemented Adobe Experience Platform 2.x SDKs for Android, then this guide will help you understand the steps required to migrate your implementation to the Experience Platform 3.x SDKs. In summary, you'll need to:
+
+1. [Update dependencies](#update-dependencies)
+2. [Update SDK initialization](#update-sdk-initialization)
+3. [Update outdated API references](#update-outdated-api-references)
+
+### Update dependencies
+
```

Comment:
I like adding a little blurb between headers since I don't like it when we have immediately back to back headers.

I put this text for the iOS-side one: "When updating to the Experience Platform 3.x SDKs, please take note of the following updates for dependencies:" (I've already updated this one for the Android-side of things.)